packages/webhooks/model-validator/test.schema.ts

```diff
@@ -0,0 +1,88 @@
+export type Maybe<T> = T | null
+export type Scalars = {
+    ID: string;
+    String: string;
+    Boolean: boolean;
+    Int: number;
+    Float: number;
+    JSON: any;
+    Upload: any;
+}
+export type File = {
+    __typename?: 'File';
+    id?: Maybe<Scalars['ID']>;
+    publicUrl?: Maybe<Scalars['String']>;
+}
+export type B2BAppAccessRight = {
+    app?: Maybe<B2BApp>;
+    id: Scalars['ID'];
+}
+export type SenderField = {
+    __typename?: 'SenderField';
+    dv: Scalars['Int'];
+    fingerprint: Scalars['String'];
+}
+export enum B2BAppGlobalFeature {
+    PropertyMapGeneration = 'PropertyMapGeneration',
+}
+
+export type B2BApp = {
```

Comment:
Why here is only B2BApp related types? Webhooks will work only for this schema?

Reply:
Nope...I've just created my own independent local schema for test purposes